web/wp-content/themes/dctx-v2/front-page.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-geujq3.php, but found phpcs-scan-GeUjQ3.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).